data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,167 @@ mission "Arachi Cartoonist 2"
+	passengers 1
+	cargo "unmanned submarines" 60
+	to offer
+		has "Coalition: Submarines 3: done"
+	source "Dusk Companion"
+	destination "Second Cerulean"
+	on offer
+		conversation
+			`Apri'ie instructs you to head to some hangars at a large port, where a team is waiting for the repair test. He hands you 130,000 credits when you land, and goes to check with the workers unloading the submarines.`
+			`	They are all taken from your cargo and swiftly put in the water as the team of Kimek boards a large ship that you guess is an old tanker. This ship prepares to follow the subs to the testing area, along with another smaller, newer ship with a thick crane on the front. Apri'ie approaches you before he boards the tanker, asking, "Come with us for the test, would you like, Captain? Take a while, it might. Not expected until after nightfall, the next tremors are."`
+			choice
+				`	"If it's alright, yeah. Could be interesting."`
+				`	"I'll pass. Not a fan of that type of ship."`
+					goto refuse
+			`	You head up the ramp to the ship, and after a few minutes it sets off, following the submarines controlled remotely by the Kimek team. Exploring the tanker, you find an odd pile of scrap metal secured as cargo on the back, the pieces welded together into a stubby cylinder about the size of the submarines. Apri'ie notices you looking at it, and comes to explain that it's the "prop" on which they'll test the repair functions.`
+			`	Once the tanker comes to a stop, some of the Kimek sailors ask everyone to clear the prop's surroundings and then attach small charges to it, detonating them once they're far enough away. The explosions are small, but make several dents on the cylinder, giving a peek of the metallic insides. The smaller ship's crane grabs the prop and slowly maneuvers it away from the tanker before lowering it into the water and letting it sink. The submarines follow the prop as it goes down, until it comes to a stop after a few minutes.`
+			`	The Kimek operators then proceed to take shifts at their computers as everyone waits for a tremor to begin. As expected, however, it is only after the moon is high in the sky that one begins. You can't see very well what is happening on the Kimek's screens, as the cameras all form a confusing set of images, like the eyes of a fly. You can, however, pinpoint the arms opening up with a small blowtorch, which, in spite of the vibrations shaking up the prop, are fairly accurate in welding back the prop's broken sections. Once they're done, and after the tremors stop, the subs slowly bring the prop back up, where the crane places it on the tanker again.`
+			`	"Let's see..." Apri'ie heads to the prop to check on the welded parts. "Yes, very good. Performed well, the welder arms have. Good for deployment, they should be." The submarines go ahead once again, lighting the way back to the port as the ships follow them until docking.`
```

Comment:
Same deal: this long and technical description of what they're doing just feels unnecessary to me. There's no real payoff to the player for learning this information, and it's not interesting or entertaining enough to stand on its own IMO.

Reply:
I still think this needs to be done. You could remove at least one full paragraph of text here and not lose anything, IMO.